src/test/words/pub-id.test.js

```diff
@@ -78,17 +78,12 @@ describe('Fix ISBN13 format\n', () => {
 
 describe('Fix ISBN number\n', () => {
 	let testCase = {
-		"978 - 80 - 902734 - 1 - 6" : "978-80-902734-1-6",
-		"978 - 80 - 902734 - 1 - 6" : "978-80-902734-1-6",
 		"978 - 80 - 902734 - 1 - 6" : "978-80-902734-1-6",
 		"978 - 80—902734—1—6" : "978-80-902734-1-6",
-		"978 - 80 - 902734 - 1 - 6" : "978-80-902734-1-6",
-		"978 - 80 - 902734 - 1 - 6" : "978-80-902734-1-6",
 		"978 - 0-9752298-0-X" : "978-0-9752298-0-X",
 		"978 - 99921 - 58 - 10 - 7" : "978-99921-58-10-7",
 		"978 - 9971 - 5 - 0210 - 0" : "978-9971-5-0210-0",
 		"978 - 960 - 425 - 059 - 0" : "978-960-425-059-0",
-		"978 - 80 - 902734 - 1 - 6" : "978-80-902734-1-6",
 		"978 - 85 - 359 - 0277 - 5" : "978-85-359-0277-5",
```

Comment:
These lines were marked as duplicates of `"978 - 80 - 902734 - 1 - 6" : "978-80-902734-1-6"`